### ISSUE TITLE
Autofill does not work for passwords

### ISSUE THREAD

**Issue by NPadrutt** [microsoft/fast#3614]
**Describe the bug; what happened?**
Usually, when I enter a username and a password in a browser the browser offers me to save it as a password, bitwarden (password manager) does the same and next time I visit the website the credentials are autofilled:

This works with, for example bootstrap inputs, but doesn't with fast-text-fields
**What are the steps to reproduce the issue?**

Example:

1. Start reproduction example
2. go to Login bootstrap page and enter some random username and password
3. See that the browser offers to safe them
4. Save and reload to see them autofilled
5. go to Fast Login page and repeat procedure
6. see that browser does not recognize the credentials

**What behavior did you expect?**
The browser should recognize my username and password, save and autofill them.

**If applicable, provide screenshots:**

![image](https://user-images.githubusercontent.com/1764367/89204544-917eec80-d5b6-11ea-9c9f-a7d31d9a7985.png)

![image](https://user-images.githubusercontent.com/1764367/89204552-9479dd00-d5b6-11ea-87f6-75ffee347d10.png)

**In what environment did you see the issue?**

* OS & Device: [e.g. MacOS, iOS, Windows, Linux] on [iPhone 7, PC] Windows
* Browser [e.g. Edge, Chrome, Safari, FireFox] Edge 
* Version [e.g. 1.8.0] 1.7 (latest on npm)

**Is there any additional context?**

**Comment by litteredwitherrors** [comment]
I've looked into this a bit. Results so far:

Edge:
- FAST: ❌
- Bootstrap: ❌

Chrome:
- FAST: ❌
- Bootstrap: ❌

Brave:
- FAST: ❌
- Bootstrap: ❌

Safari:
- FAST: ✅ 
- Bootstrap: ✅ 

Firefox:
- FAST: ✅ 
- Bootstrap: ✅

Still trying to figure out why Save Password is not being triggered in Chromium based browsers. I might have to attempt with a different app.

References:
https://developers.google.com/web/fundamentals/security/credential-management/save-forms#full_code_example
https://stackoverflow.com/questions/2382329/how-can-i-get-browser-to-prompt-to-save-password

**Comment by chrisdholt** [comment]
Thanks for the update @litteredwitherrors! @EisenbergEffect as FYI - looks like this is primarily an issue with Chromium based browsers...

**Comment by NPadrutt** [comment]
@litteredwitherrors bootstrap had the same issues in your tests you say? For me that seemed to work as expected in Chrome and Edge Chromium.

**Comment by litteredwitherrors** [comment]
@NPadrutt Yes, I noticed in your example Bootstrap did work. For one reason or another I have not been able to reproduce that in any Chromium browser.

**Comment by NPadrutt** [comment]
I see, interessting.

**Comment by NPadrutt** [comment]
Hi!
I wanted to check in if there is any update on this issue?

**Comment by janechu** [comment]
`@microsoft/fast-components` has been deprecated some time ago, closing this issue.
